.taskcluster.yml

```diff
@@ -146,8 +146,8 @@ tasks:
               in:
                   $if: >
                       tasks_for in ["action", "cron"]
-                      || (isPullRequest && pullRequestAction in ["opened", "reopened", "synchronize"])
-                      || (tasks_for == "github-push" && head_ref[:10] != "refs/tags/" && short_head_ref != "staging.tmp" && short_head_ref != "trying.tmp" && short_head_ref[:8] != "mergify/")
+                      || (isPullRequest && pullRequestAction in ["opened", "reopened", "synchronize"] && baseRepoUrl != repoUrl)
```

Comment:
Reminder (because I'm the type of person who forgets that stuff 😅): Run [ci-config](https://hg.mozilla.org/ci/ci-configuration/) after this PR reaches the `main` branch.